queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```

Comment:
- [ ] The second query is not running properly and not giving results, as I can see in the screenshot. Please make sure that it runs without any issues and produces the expected results.
  - You can also try with `SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;`.

![image](https://user-images.githubusercontent.com/83803414/208712709-0cfc53bd-683e-4881-b891-816f1e389681.png)

Reply:
Hi, @thecodechaser !!!
That line was an error. The second query is actually after that one. It is the one in the queries.sql file: 
SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31'.

Reply:
@Samuellekpor Ohh, I didn't notice that. Please ignore this change and only work on the README change as I requested and go for another review.

Reply:
Alright! Thank you!!